### PR TITLE
Code cov

### PR DESCRIPTION
**Pull request by jjeane**
### What is the context of this PR?

Added codecov to survey runner
Travis can be set up to fail under certain code coverage %
### How to review

Ensure codecov is running on this branch
In travis.yml uncomment out

`- coverage report --fail-under=80` 

and change value to ensure that build fails/passes as per the criteria
- [ ] Do all commits have sensible commit messages?
- [ ] Is all new code unit tested?
- [ ] Are there integration tests if needed?
- [ ] Is there sufficient logging?
- [ ] Is there documentation or is the code self-describing?
### Who worked on the PR

@jjeane

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ python:
 - '3.5'
 env:
 - EQ_SCHEMA_BUCKET=""
+before_install:
+  - pip install codecov
```
Comment (warrenbailey) [inline .travis.yml]: This dependency could be added to the requirements.txt instead

File: .travis.yml [hunk 2]
```diff
@@ -19,6 +21,8 @@ before_script:
 - "./scripts/run_app.sh"
 script:
 - "./scripts/run_tests.sh"
+# fail travis if under X% test coverage
+# - coverage report --fail-under=80
```
Comment (warrenbailey) [inline .travis.yml]: Might as well enable this, but set it to the around current code coverage level, so 77%?

At least that makes sure people don't make it worse, we can always up it to 80% when we get there.

**Comment by warrenbailey** [comment]
A few minor changes and then this is good to go. Also needs rebasing to master.

I like this code coverage tool :)

**Comment by dhilton** [comment]
@warrenbailey Are you ok to make those changes you've mentioned? They all look sensible amendments (IMHO). So looks good to merge 👍

**Comment by warrenbailey** [comment]
👍

**Pull request merged**